nexus/tests/output/uncovered-authz-endpoints.txt

```diff
@@ -1,22 +1,36 @@
 API endpoints with no coverage in authz tests:
+organization_delete                      (delete "/organizations/{organization_name}")
+project_delete                           (delete "/organizations/{organization_name}/projects/{project_name}")
+instance_delete                          (delete "/organizations/{organization_name}/projects/{project_name}/instances/{instance_name}")
+instance_view_by_id                      (get    "/by-id/instances/{id}")
+organization_view_by_id                  (get    "/by-id/organizations/{id}")
+project_view_by_id                       (get    "/by-id/projects/{id}")
 login_saml_begin                         (get    "/login/{silo_name}/saml/{provider_name}")
+organization_list                        (get    "/organizations")
+organization_view                        (get    "/organizations/{organization_name}")
+organization_policy_view                 (get    "/organizations/{organization_name}/policy")
+project_list                             (get    "/organizations/{organization_name}/projects")
+project_view                             (get    "/organizations/{organization_name}/projects/{project_name}")
+instance_list                            (get    "/organizations/{organization_name}/projects/{project_name}/instances")
+instance_view                            (get    "/organizations/{organization_name}/projects/{project_name}/instances/{instance_name}")
+instance_serial_console                  (get    "/organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/serial-console")
+instance_serial_console_stream           (get    "/organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/serial-console/stream")
+project_policy_view                      (get    "/organizations/{organization_name}/projects/{project_name}/policy")
```

Comment:
There's work I could do here to get this test passing for all the old endpoints but I'm opting _not_ to do that in favor of making process. The authz test is non-trivial and I'm trying to update it as little as possible until I can get a critical mass of APIs converted. I _am_ going through and ensuring the new API endpoints are correctly tested though. 

On that note, there is a slight issue in that you can only test and endpoint from the spec once. If you try to verify it more than once it freaks out on you. I don't have the exact error handy but it does this because it "consumes" routes from the spec and if it encounters something its already consumed then that's considered a failure. We actually need to be able to test a route at least twice (name path, id). Anyway, I'll add that to the top level issue to resolve.